webcompat/form.py

```diff
@@ -428,15 +444,20 @@ def build_formdata(form_object):
     if extra_labels:
         metadata_keys.append('extra_labels')
 
+    clean = re.compile('<.*?>')
```

Comment:
What is this regex trying to solve? regex and html are not good friends.

Reply:
So as [these options](https://www.screencast.com/t/kdoYgXDiKhV) contain HTML, this regex is eliminating all the tags, leaving only the text for the options. @johngian if you know a better option, I would appreciate your help.

Reply:
[Linking to the code](https://github.com/webcompat/webcompat.com/blob/d995b67769fb0b5bb4bed3a529dea2f2a315a43e/webcompat/form.py#L82-L121) to have a resilient trace of what we are talking about